packages/transport/src/modules/accounts/IdentityDeletionProcessController.ts

```diff
@@ -0,0 +1,90 @@
+import { log } from "@js-soft/ts-utils";
+import { ControllerName, TransportController } from "../../core";
+import { IdentityDeletionProcessStatusChangedEvent } from "../../events";
+import { AccountController } from "../accounts/AccountController";
+import { SynchronizedCollection } from "../sync/SynchronizedCollection";
+import { IdentityDeletionProcessClient } from "./backbone/IdentityDeletionProcessClient";
+import { IdentityDeletionProcess } from "./data/IdentityDeletionProcess";
+import { IdentityDeletionProcessStatus } from "./data/IdentityDeletionProcessStatus";
+
+export class IdentityDeletionProcessController extends TransportController {
+    public identityDeletionProcessClient: IdentityDeletionProcessClient;
+    public identityDeletionProcessCollection: SynchronizedCollection;
+
+    public constructor(parent: AccountController) {
+        super(ControllerName.Identity, parent);
+
+        this.identityDeletionProcessClient = new IdentityDeletionProcessClient(this.config, this.parent.authenticator);
+    }
+
+    @log()
+    public override async init(): Promise<IdentityDeletionProcessController> {
+        await super.init();
+
+        this.identityDeletionProcessCollection = await this.parent.getSynchronizedCollection("IdentityDeletionProcesses");
+        return this;
+    }
+
+    public async getIdentityDeletionProcess(identityDeletionProcessId: string): Promise<IdentityDeletionProcess | undefined> {
+        const identityDeletionProcess = await this.identityDeletionProcessCollection.findOne({ id: identityDeletionProcessId });
+        return identityDeletionProcess ? IdentityDeletionProcess.from(identityDeletionProcess) : undefined;
+    }
+
+    public async getIdentityDeletionProcesses(): Promise<IdentityDeletionProcess[]> {
+        return (await this.identityDeletionProcessCollection.find())
+            .map((identityDeletionProcess) => (identityDeletionProcess ? IdentityDeletionProcess.from(identityDeletionProcess) : undefined))
+            .filter((identityDeletionProcess) => !!identityDeletionProcess) as IdentityDeletionProcess[];
+    }
+
+    public async getApprovedIdentityDeletionProcess(): Promise<IdentityDeletionProcess | undefined> {
+        const identityDeletionProcess = await this.identityDeletionProcessCollection.findOne({
+            status: IdentityDeletionProcessStatus.Approved
+        });
+        return identityDeletionProcess ? IdentityDeletionProcess.from(identityDeletionProcess) : undefined;
+    }
+
+    public async getWaitingIdentityDeletionProcess(): Promise<IdentityDeletionProcess | undefined> {
+        const identityDeletionProcess = await this.identityDeletionProcessCollection.findOne({
+            status: IdentityDeletionProcessStatus.WaitingForApproval
+        });
+
+        return identityDeletionProcess ? IdentityDeletionProcess.from(identityDeletionProcess) : undefined;
+    }
```

Comment:
I would vote for just one method that let's you pass a IdentityDeletionProcessStatus. Maybe allow more than one status by spread operator, would be easy to simplify the part in `packages/runtime/src/useCases/transport/identityDeletionProcesses/GetIdentityDeletionProcess.ts` where you query two times which could be one operation.